app/models/program.rb

```diff
@@ -87,6 +91,17 @@ def languages_list
     self.languages.split(',').map {|l| ISO_639.find(l).english_name} if self.languages.present?
   end
 
+  ##
+  # Checks if there is any event in the program that starts in the given date
+  #
+  # ====Returns
+  # * +True+ -> If there is any event for the given date
+  # * +False+ -> If there is not any event for the given date
+  def any_event_for_this_date?(date)
+    parsed_date = DateTime.parse("#{date} 00:00").utc
+    events.where(start_time: parsed_date..(parsed_date + 1)).any?
```

Comment:
Can they? The `start_time` is given in the admin schedule in the backend and all event are saved without timezone, that is with the default one which is UTC. :confused: